### PR TITLE
fix: Add chattr +remove support for files

### PR DESCRIPTION
**Pull request by halostatue**
Resolves: #3282

### CODE REVIEW

**Comment by halostatue** [comment]
I should probably modify this PR and the commit, because `+remove` is already supported for directories, but not files.

File: internal/cmd/testdata/scripts/chattr.txtar [hunk 1]
```diff
@@ -6,6 +6,16 @@ exec chezmoi chattr empty $HOME${/}.file
 ! exists $CHEZMOISOURCEDIR/dot_file
 exists $CHEZMOISOURCEDIR/empty_dot_file
 
+# test that chezmoi chattr remove sets the remove attribute on a file
+exec chezmoi chattr remove $HOME${/}.file
+! exists $CHEZMOISOURCEDIR/empty_dot_file
+exists $CHEZMOISOURCEDIR/remove_dot_file
+
+# test that chezmoi chattr noremove sets the remove attribute on a file
```
Comment (twpayne) [inline internal/cmd/testdata/scripts/chattr.txtar:14]: nit: s/sets the remove attribute/removes the remove attribute/

**Review by twpayne — APPROVED** [review]
Thank you for fixing this :)

File: internal/cmd/testdata/scripts/chattr.txtar (hunk 1 above)
Comment (halostatue) [reply internal/cmd/testdata/scripts/chattr.txtar:14]: I’ll fix this this evening.

**Pull request merged**